File: Kotlin/URI_1011/Volume_Esfera_1011.kt
import kotlin.math.pow
fun Vol_Esfera(r:Double):Double = (4/3.0) * 3.14159 * r.pow(3)
fun main(arg:Array<String>){
    val raio:Double = readLine()!!.toDouble()
    println("VOLUME = %.3f".format(Vol_Esfera((raio))))
}

File: C#/URI_1011/Volume_Esfera_1011.cs

using System;
    
class Program{
    static void Main(){

    double r = Double.Parse(Console.ReadLine());

    Func <double,double> Volume_Esfera = raio => 3.14159 * 4/3 *raio*raio*raio;
    
    Console.WriteLine (Math.Round(Volume_Esfera(r),3));

        }
    
}